test/start.test.js

```diff
@@ -556,7 +556,7 @@ test('should start the server listening on 0.0.0.0 when running in kubernetes',
   t.pass('server closed')
 })
 
-test('should start the server with watch options that the child process restart when directory changed', { skip: process.platform === 'win32' }, async (t) => {
+test('should start the server with watch options that the child process restart when directory changed', { skip: process.platform === 'win32' || (process.platform === 'darwin' && process.version.startsWith('v20')) }, async (t) => {
```

Comment:
I'm not sure why these tests are failing, but this is the slimmest skip I can think of.